modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/MoveKBObjectMVCActionCommand.java

```diff
@@ -140,13 +149,31 @@ protected void doProcessAction(
 			}
 		}
 		catch (PortalException portalException) {
-			if (portalException instanceof DuplicateLockException) {
+			if (portalException instanceof LockedKBArticleException) {
+				LockedKBArticleException lockedKBArticleException =
+					(LockedKBArticleException)portalException;
+
```

Comment:
Cannot we use a catch block for LockedKBArticleException? It shares no logic with the generic PortalException.

Reply:
I agree